app/assets/stylesheets/products.scss

```diff
@@ -0,0 +1,81 @@
+// Place all the styles related to the products controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+
+.parallax-1 {
```

Comment:
Each of the parallax styles are the same except for the image. You should be using one general class that includes all of the common parallax features, and then add another class for each specific image.